docformatter.py

```diff
@@ -738,6 +741,10 @@ def _format_files(args, standard_out, standard_error):
         FormatResult.check_failed,
         FormatResult.ok,
     ]
+
+    if args.exit:
+        return_codes.insert(2, FormatResult.ok_file_modified)
```

Comment:
Rather than inserting, can you do something like the following to avoid the magic number of 2?

```

    return_codes = [  # in order of preference
        FormatResult.error,
         FormatResult.check_failed,
    ]

    if args.exit:
        return_codes.append(FormatResult.ok_file_modified)

    return_codes.append(FormatResult.ok)
```

Reply:
Welcome back! Thanks for the review, I made the suggested changes.